src/AspNet.Security.OAuth.AdobeIO/AdobeIOAuthenticationOptions.cs

```diff
@@ -24,8 +24,8 @@ public AdobeIOAuthenticationOptions()
             AuthorizationEndpoint = AdobeIOAuthenticationDefaults.AuthorizationEndpoint;
             TokenEndpoint = AdobeIOAuthenticationDefaults.TokenEndpoint;
             UserInformationEndpoint = AdobeIOAuthenticationDefaults.UserInformationEndpoint;
-            Scope.Clear();
-            Scope.Add("openid,AdobeID");
+            Scope.Add("openid");
+            Scope.Add("AdobeID");
             ClaimActions.MapJsonKey(ClaimTypes.NameIdentifier, "sub");
```

Comment:
Minor refactor for readability.

```suggestion

            Scope.Add("openid");
            Scope.Add("AdobeID");

            ClaimActions.MapJsonKey(ClaimTypes.NameIdentifier, "sub");
```

Reply:
Done